src/chi/components/sidebar/sidenav.scss

```diff
@@ -887,6 +897,8 @@ $sidenav-lg-width: 12.5rem;
 
                 .chi-link {
                   display: flex;
+                  margin-bottom: 0.125rem;
+                  margin-top: 0.125rem;
```

Comment:
@ASanchLumen @josecarlosgonzalezv didn't we talk about adding this rule to chi-link?